Platform/Intel/MinPlatformPkg/Acpi/AcpiTables/AcpiPlatform.c

```diff
@@ -197,6 +297,11 @@ CreateCpuLocalApicInTable (
     }
   }
 
+  //
+  // Perform quick sort
+  //
+  QuickSort (CpuApicIdOrderTable, mNumberOfCpus, sizeof (EFI_CPU_ID_ORDER_MAP), (BASE_SORT_COMPARE)ApicIdOrderTableCompare, &SortBuffer);

```

Comment:
The ApicIdOrderTableCompare () is too complex.
How about?
1. do an insert-sort to move big cores in front of small cores.
2. Move BSP in the beginning.

Reply:
Acctually I moved BSP in the beginning first and then perform the insert-sort for the rest cores.
If perform step1 first, the Bsp may be moved to another position. Another for-loop is needed to find where BSP is.